components/Create/CreateContainer.js

```diff
@@ -0,0 +1,93 @@
+import {Icon} from "@iconify/react";
+import {useState, useEffect} from "react";
+import HorizontalScroll from "../Explore/HorizontalScroll";
+import CreateExercisesContainer from "./CreateExerciseContainer";
+import {Container} from "../../style/Create/createStyling";
+import {SearchContainer} from "../../style/Create/createStyling";
+import {SearchBar} from "../../style/Create/createStyling";
+import {SearchButton} from "../../style/Create/createStyling";
+import {exerciseOptions, fetchData} from "../Utils/fetchData";
+
+const CreateContainer = ({selectedExercises, toggleExercise}) => {
+  /*------------------------STATES----------------------------*/
+  const [search, setSearch] = useState("");
+  const [bodyParts, setBodyParts] = useState([]);
+  const [exercises, setExercises] = useState([]);
+  const [bodyPart, setBodyPart] = useState("all");
+
+  /*------------------------FETCH BODYPART LIST-------------------------*/
+  useEffect(() => {
+    const fetchExercisesData = async () => {
+      const bodyPartsData = await fetchData(
+        "https://exercisedb.p.rapidapi.com/exercises/bodyPartList",
+        exerciseOptions
+      );
+
+      setBodyParts(["all", ...bodyPartsData]);
+    };
+
+    fetchExercisesData();
+  }, []);
```

Comment:
You should wrap your async code into a `try / catch` block. Fetching data can always fail (no server, no internet) and you should react appropriately.